CMakeLists.txt

```diff
@@ -8,6 +14,29 @@ set(MAJOR_VERSION "9")
 set(MINOR_VERSION "2")
 set(PATCH_VERSION "5")
 
+IF (WIN32)
```

Comment:
Please use lower case here: `if(WIN32)`

Can we move this to a dedicated cmake file? Like _configure/win.cmake_ and then just include it.